.github/workflows/test.yml

```diff
@@ -69,6 +69,13 @@ jobs:
       if: matrix.rootless != 'rootless'
       run: sudo -E PATH="$PATH" -- make localunittest
 
+    # We need to continue support for 32-bit ARM.
+    # However, we do not have 32-bit ARM CI, so we use i386 for testing 32bit stuff.
+    # We are not interested in providing official support for i386.
+    - name: unit test (i386)
+      if: matrix.rootless != 'rootless'
+      run: sudo -E PATH="$PATH" -- make GOARCH=386 localunittest
```

Comment:
the nsenter test doesn't work in cross build. In cross build, cgo is disabled, so `nsenter_unsupported.go` is used.

However it should work if running the test in docker, with `--platform=386` args.

```
$ docker run --privileged -it --rm --platform 386 -v `pwd`:/work golang:1.15-buster bash
root@3084de9b4c8f:/go# cd /work/libcontainer/nsenter
root@3084de9b4c8f:/work/libcontainer/nsenter# go test -v .
=== RUN   TestNsenterValidPaths
--- PASS: TestNsenterValidPaths (0.00s)
=== RUN   TestNsenterInvalidPaths
--- PASS: TestNsenterInvalidPaths (0.00s)
=== RUN   TestNsenterIncorrectPathType
--- PASS: TestNsenterIncorrectPathType (0.00s)
=== RUN   TestNsenterChildLogging
--- PASS: TestNsenterChildLogging (0.00s)
PASS
ok      github.com/opencontainers/runc/libcontainer/nsenter     0.012s
```

Another way it seems cgo can be manually enabled with `CGO_ENABLED=1` env, but need to figure out how to setup gcc cross build env.

Reply:
Thanks a lot @zhsj, I made it running!